prototypes/5.ai-steer/LAI.gd

```diff
@@ -0,0 +1,69 @@
+extends Node
+"""
+Sample AI controller. It gets and sets the player in its `_ready` function as a target to various
+behaviours, and picks whichever one it should be calculating on. It also updates the controller
+with information for the behaviours to use. Then, based on what is calculated, it applies
+the calculated velocity to itself. Mostly a playground.
+"""
+
+var _target: Node2D
+var _motion: = SteeringMotion2D.new()
+
+var _controller: BehaviourController2D
+var _behaviour: SteeringBehaviour2D
+var _arrive: ArriveBehaviour2D
+var _align: AlignBehaviour2D
+var _priority: PrioritySteering2D
+var _flee: FleeBehaviour2D
+
+var _velocity: = Vector2()
+var _rotation_velocity: float
```

Comment:
Could you use `angular_velocity` to stay consistent with Godot's naming? (RigidBody nodes, particles)